public/app/index.html

```
@@ -399,46 +399,92 @@ <h1 class="hdr-title">VoiceIsolate Pro <span class="hdr-ver">v24.0</span></h1>
+          <div class="iso-section-hdr">2 · Mute Background Sounds
+            <span style="font-size:0.55rem;font-weight:400;color:#64748b;margin-left:6px;">Applied during Process</span>
           </div>
-          <div class="sr-row" data-desc="Applies post-processing refinement to the spectral separation mask. Smooths edges between voice and noise regions, reducing artifacts like musical noise and spectral tearing. Disable for raw unprocessed masks.">
-            <label class="sr-label" for="isolationMaskRefine">Mask Refine</label>
-            <label style="display:inline-flex;align-items:center;gap:6px;cursor:pointer;">
-              <input type="checkbox" id="isolationMaskRefine" checked />
-              <span style="font-size:11px;color:#9ca3af;">Enabled</span>
-            </label>
+          <p class="iso-hint">Toggle sounds you want suppressed from the output. Works alongside noise reduction.</p>
+          <div class="sound-mute-grid" id="soundMuteGrid">
+            <button class="sound-mute-btn" data-cat="appliance" title="Washing machine, AC, fan, dryer — low-frequency mechanical hum (20–500 Hz)">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)